compositor_pipeline/src/error.rs

```diff
@@ -80,13 +92,18 @@ impl PipelineErrorInfo {
 }
 
 const INPUT_STREAM_ALREADY_REGISTERED: &str = "INPUT_STREAM_ALREADY_REGISTERED";
+const DECODER_ERROR: &str = "INPUT_STREAM_DECODER_ERROR";
```

Comment:
Can we detect port already in use error?

Reply:
This would be useful to search for available ports on compositor side #227

Reply:
Yes, unfortunately it's ugly ;(